src/gmp.c

```diff
@@ -8674,13 +8674,6 @@ buffer_config_preference_xml (GString *buffer, iterator_t *prefs,
       char *pos = strchr (value, ';');
       if (pos) *pos = '\0';
       buffer_xml_append_printf (buffer, "<value>%s</value>", value);
-      while (pos)
```

Comment:
Surely this hasn't been broken since 2010?  I think this worked because OTP sent the alts with the value.  Has ospd-openvas stopped doing that?

Reply:
Hmm, I think somewhere along the line we lost track of keeping the alts in the preference value in the XML.  For example in the feed config XML one radio value includes the alts but the other one does not.

And gvmd was not checking for the alts when importing XML configs, so the values ended up in dbs without alts.  My current db has both.

I think using using default_value which comes from the nvts table is a good solution.